Doc/library/intro.rst

```diff
@@ -64,3 +64,50 @@ Notes on availability
   libc version, then both conditions must hold. For example a feature with note
   *Availability: Linux >= 3.17 with glibc >= 2.27* requires both Linux 3.17 or
   newer and glibc 2.27 or newer.
+
+.. _wasm-availability:
+
+Notes on WebAssembly platforms
+==============================
+
+The WebAssembly platforms ``wasm32-emscripten`` and ``wasm32-wasi`` provide a
+subset of POSIX APIs. WASM runtimes and browser are sandboxed and have
+limited access to the host and external resources. Any standard library module
+that uses processes, threading, networking (:mod:`socket`), signals, or
+other forms of inter-process communication (IPC), is either not available
+or may not work as on other Unix-like systems. File I/O, file system, and Unix
+permission-related functions are restricted, too. Emscripten does not permit
+blocking operations like blocking network I/O and :func:`~time.sleep`.
+
+The properties and behavior of Python on WebAssembly platforms depend on the
+`Emscripten`_ or `WASI`_-SDK version, WASM runtimes (browser, NodeJS, `wasmtime`_),
+and Python build time flags. WebAssembly, Emscripten, and WASI are evolving.
+Some features like networking may be supported in the future.
+
+For Python in the browser, users should consider `Pyodide`_ or `PyScript`_.
+PyScript is built on top of Pyodide, which itself is built on top of
+CPython and Emscripten. Pyodide provides access to browsers' JavaScript and
+DOM APIs as well as limited networking capabilities with JavaScript's
+``XMLHttpRequest`` and ``Fetch`` API.
```

Comment:
Looking forward to the day when that's just a "browser support module" installable in upstream CPython...

Reply:
I'm lobbying for a high level HTTP API in Python core.